src/main/scala/main.scala

```diff
@@ -127,6 +127,89 @@ object main extends App{
 
   println(iteration3("Fizz","Buzz",15))
   */
+  /*
+  def recursion1(word:String, num:Int){
+    if(num>0){
+      println(word)
+      recursion1(word, num-1)
+    }
+  }
+  recursion1("Hi",4)
+  */
+  /*
+  def recursion2(word:String, num:Int){
+    val lenWord = word.length
+    val line =word*4
+    if(num>0){
+      println(line)
+      recursion2(word,num-1)
+    }
+  }
+  recursion2("H",7)
+  */
+
+  /* //I haven't quite got this exercise to work properly yet. In its current state it outputs ",Buzz".
+  var output=""
+  def recursion3(f:String, b:String, num:Int):String={
```

Comment:
that's fine although i'd suggest making it with pattern matching rather than ifs